config/examples/settings.yml

```diff
@@ -5,7 +5,6 @@ base: &default
   force_ssl: <%= ENV.fetch('FORCE_SSL', '1') == '1' %>
   apicast_oauth: <%= ENV.fetch('ENABLE_APICAST_OAUTH', '0') == '1' %>
   apicast_custom_url: <%= ENV.fetch('ENABLE_APICAST_CUSTOM_URL', '0') == '1' %>
-  active_docs_url: <%= ENV.fetch('ACTIVE_DOCS_URL', "https://support-staging.#{superdomain}") %>
```

Comment:
The usage was removed here: https://github.com/3scale/system/commit/90ebb558ef168550289c17dc8b3a4123cd3f2987#diff-67dc91a359da89ceb6c89092f4797e95a1ba6c16a5b915de4a560ae68e61a56f